<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LAB_12_14Class</class>
 <widget class="QMainWindow" name="LAB_12_14Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LAB_12_14</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="MyTabWidget">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>-31</y>
      <width>591</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="IntroTab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string> What are you ?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SendToAdminTab">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ADMIN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SendToUserTab">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>120</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>USER</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="AdminTab">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QListWidget" name="AdminListWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>331</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="RemoveButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>40</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Delete this cat</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NameLine">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>239</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="BreedLine">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>271</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Breed</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="AgeBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>300</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Weight</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="WeightBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="minimum">
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>360</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Link</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LinkLine">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>360</y>
        <width>471</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="AddButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>240</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add this cat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UpdateButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>300</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Update this cat</string>
      </property>
     </widget>
     <widget class="QLabel" name="ErrorLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Go back</string>
      </property>
     </widget>
     <widget class="QComboBox" name="MyComboBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Detailed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Short</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="UserTab">
     <attribute name="title">
      <string>Tab 3</string>
     </attribute>
     <widget class="QListWidget" name="AdoptionListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>261</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Go back</string>
      </property>
     </widget>
     <widget class="QLabel" name="AdoptionLabel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="AdoptButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>130</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Adopt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NextButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>130</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="FilterBreed">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QSpinBox" name="WeightFilter">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>310</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
     </widget>
     <widget class="QPushButton" name="AdoptButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Adopt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NextButton_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>360</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLabel" name="FilterLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="FilterButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>310</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CSVButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>CSV</string>
      </property>
     </widget>
     <widget class="QPushButton" name="HTMLButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>HTML</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LAB_12_14.qrc"/>
 </resources>
 <connections/>
</ui>
